helm/ffc-pay-enrichment/values.yaml

```diff
@@ -19,17 +19,12 @@ containerSecret:
   type: Opaque
 
 deployment:
-  restartPolicy: Always
-  priorityClassName: default
-  replicas: 1
   minReplicas: 1
   maxReplicas: 16
   cpuThreshold: 80
   memoryThreshold: 80
-  minReadySeconds: 0
 
 container:
```

Comment:
And here: https://github.com/DEFRA/ffc-helm-library/blob/master/ffc-helm-library/templates/_container.yaml